tools/tpm2_getcap.c

```diff
@@ -801,6 +834,9 @@ static bool on_option(char key, char *value) {
     switch (key) {
     case 'l':
         options.list = true;
+        break;
+    case 'i':
+        options.ignore_more = true;
```

Comment:
Elsewhere in all the tools `-i` is regarded as input. So maybe drop the short option.

Reply:
I felt dirty about using -i, but it didn't even occur to me to just keep the long option.